.changeset/twenty-pigs-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'skuba': minor
+---
+
+**configure, init:** Format `package.json` with [sort-package-json](https://github.com/keithamus/sort-package-json)
```

Comment:
I think this will be bolded automatically: https://github.com/seek-oss/skuba/blob/3d9fa47704bb1a6e8397347c46f1cc27b3b6b709/.changeset/changelog.js#L13

```suggestion
configure, init: Format `package.json` with [sort-package-json](https://github.com/keithamus/sort-package-json)
```